### DISCUSSION TITLE
Add the :joy: emoji as a reaction option

### DISCUSSION THREAD

**Discussion by daleal** [community/community#106799] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I know it feels like a small suggestion, but I really think that this would have tangible improvements over the expression of sentiment throughout the GitHub experience for teams of any size. There is a 😄 emoji available, but it just doesn't communicate the same. Anyone knows if there is a specific reason why the 😂 emoji can't be used to react to PR comments and stuff? Note here that I am not suggesting adding **every** available emoji, or even custom emojis (which would be **awesome**, btw), I just find myself missing that particular emoji so damn much.

**Comment by danmoseley** [comment]
I'd like the option of all emojis. The type of the emoji is already exposed in the API as a string (rather than special casing the existing few)

**Reply by Blaumaus** [reply]
I guess they won't add all emojis, because some of them may be used for stuff like bullying with emojis like 💩, 💀, 🤮, etc.
It would be nice if they let repo maintainers to pick what emojis should be available for issues and PRs

**Reply by daleal** [reply]
I only proposed that one emoji because I imagine that adding every emoji would need some UX work (how to add them to the UI, the search component, what to do when too much emojis are used to react, etc). I think that adding just that one emoji should be reasonably simple, and in my opinion would be a really good improvement.

**Comment by atharva5683** [comment]
😂😂
